mir_dataset_loaders/load_utils.py

```diff
@@ -0,0 +1,40 @@
+from collections import namedtuple
+import os
+
+from . import MIR_DATASETS_DIR
+
+
+def get_local_path(rel_path, data_home):
```

Comment:
nitpicky: swap these arguments around to get a more intuitive function `data_home, rel_path => data_home/rel_path`